src/openloco/ui/WindowManager.h

```diff
@@ -154,7 +157,6 @@ namespace openloco::ui::windows::ScenarioOptions
 namespace openloco::ui::windows::station
 {
     window* open(uint16_t id);
-    void tab_2_scroll_paint(window& w, gfx::drawpixelinfo_t& dpi);
```

Comment:
Nice catch. I wonder why our CI didn't complain when it was renamed and made static in #441…